Yuzu requires a version of Vulkan that has not yet been released as a stable Vulkan SDK. In case this patch fails, check which version Yuzu is currently using and verify that it still works with the version shipped in Nixpkgs.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -314,7 +314,7 @@
 find_package(zstd 1.5 REQUIRED)
 
 if (NOT YUZU_USE_EXTERNAL_VULKAN_HEADERS)
-    find_package(Vulkan 1.3.256 REQUIRED)
+    find_package(Vulkan 1.3.250 REQUIRED)
 endif()
 
 if (ENABLE_LIBUSB)
